src/cursor.c

```diff
@@ -891,22 +891,34 @@ handle_cursor_axis(struct server *server, struct cursor_context *ctx,
 	uint32_t modifiers = wlr_keyboard_get_modifiers(
 			&server->seat.keyboard_group->keyboard);
 
-	uint32_t button = 0;
-	if (event->delta_discrete < 0) {
-		button = BTN_GEAR_UP;
-	} else if (event->delta_discrete > 0) {
-		button = BTN_GEAR_DOWN;
+	bool found_direction = false;
```

Comment:
If we use our own enum here the bool isn't required anymore and we could just initialize the enum with `LAB_DIRECTION_INVALID` + check for it below.